docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -2078,52 +669,6 @@ Transforms an array into rows.
 +-----------------------------------+
 ```
 
-### `range`
-
-Returns an Arrow array between start and stop with step. `SELECT range(2, 10, 3) -> [2, 5, 8]` or `SELECT range(DATE '1992-09-01', DATE '1993-03-01', INTERVAL '1' MONTH);`
-
-The range start..end contains all values with start <= x < end. It is empty if start >= end.
-
-Step can not be 0 (then the range will be nonsense.).
-
-Note that when the required range is a number, it accepts (stop), (start, stop), and (start, stop, step) as parameters, but when the required range is a date or timestamp, it must be 3 non-NULL parameters.
-For example,
-
-```
-SELECT range(3);
-SELECT range(1,5);
-SELECT range(1,5,1);
-```
-
-are allowed in number ranges
-
-but in date and timestamp ranges, only
-
-```
-SELECT range(DATE '1992-09-01', DATE '1993-03-01', INTERVAL '1' MONTH);
-SELECT range(TIMESTAMP '1992-09-01', TIMESTAMP '1993-03-01', INTERVAL '1' MONTH);
-```
-
-is allowed, and
-
-```
-SELECT range(DATE '1992-09-01', DATE '1993-03-01', NULL);
-SELECT range(NULL, DATE '1993-03-01', INTERVAL '1' MONTH);
-SELECT range(DATE '1992-09-01', NULL, INTERVAL '1' MONTH);
-```
-
-are not allowed
-
-#### Arguments
-
-- **start**: start of the range. Ints, timestamps, dates or string types that can be coerced to Date32 are supported.
-- **end**: end of the range (not included). Type must be the same as start.
-- **step**: increase by step (can not be 0). Steps less than a day are supported only for timestamp ranges.
-
-#### Aliases
-
-- generate_series
-
 ## Struct Functions
 
 - [unnest](#unnest-struct)
```

Comment:
I think `unnest` is special and not a scalar function. Perhaps as a follow on PR we can make a new page called "Special Functions" and put it there.

Reply:
Yeah, I can create one for that

Reply:
The only other special function I know of is make_map. See https://github.com/apache/datafusion/blob/972e3abea4286b0d06c44498d576c8498ddd3be2/datafusion/sql/src/expr/function.rs#L229

Reply:
Filed as https://github.com/apache/datafusion/issues/13036